### PR TITLE
Feature/list-tracks

### PR DESCRIPTION
**Pull request by hugurp**
- active track in list
- animation on list start

### CODE REVIEW

File: src/PlayerData.vue [hunk 1]
```diff
@@ -41,15 +49,21 @@
...
 			}
 		},
 		created() {
-			// init 1 - from what `key` do we load channel data?
+			// init 1 - if there is any following key,
+			// start the according session
+			// otherwise stay in idle mode
 			const { channelSlug, channelId, trackId } = this;
+			if (!channelSlug && !channelId && !trackId ) {
+				this.playerStartedSession = false;
```
Comment (oskarrough) [inline src/PlayerData.vue]: @hugurp since ``playerStartedSession` is false by default, does this do anything?

**Comment by oskarrough** [comment]
@hugurp Loading is nice now! One more thing, tracks are no longer newest on top. 

Perhaps we just need to do reverse when setting trackspool? `this.tracksPool = tracks.reverse();`

File: src/PlayerData.vue (hunk 1 above)
Comment (hugurp) [inline src/PlayerData.vue]: not used anymore, will remove that.

Comment (hugurp) [inline src/PlayerData.vue]: ah no wait, it is used in `startR4Session`

Comment (hugurp) [inline src/PlayerData.vue]: try to load a player with radio or track

**Pull request merged**